zuul.d/projects.yaml

```
@@ -74,12 +74,4 @@
     github-check:
       jobs:
         - telemetry-operator-multinode-autoscaling-tempest
```

```suggestion
        - telemetry-operator-multinode-autoscaling-tempest:
            required-projects:
              - name: openstack/telemetry-tempest-plugin
                override-checkout: master
```
The connection may need to be stated. i.e. opendev.org/openstack/telemetry-tempest-plugin
https://review.rdoproject.org/r/plugins/gitiles/config/+/refs/heads/master/zuul/rdo.yaml#1236